vignettes/E01_simple-example.Rmd

```diff
@@ -23,14 +23,15 @@ library(ggplot2)
 library(latex2exp)
```

Comment:
Are any of these changes of particular interest to us?

Reply:
It looks like most of the changes are additions to the analysis and clarifications to the explanations, so it might be nice to incorperate them